src/server/server.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args));
+
+const proxy = require('express-http-proxy'),
+  app = require('express')(),
```

Comment:
please declare all variables separately and the funcs separately. It's difficult to read

Reply:
did you forget about that?

Reply:
you mean that variables need to be at the top of file and functions below?

Reply:
you have 1 const for 3 variables and you're calling the app one on top of that. Please separate those. Your number 1 priority here should be readability.